doc/ref_arch/openstack/chapters/chapter06.md

```diff
@@ -204,6 +204,35 @@ Operators typically do not implement Security Groups when ussing SR-IOV or DPDK
 
 <a name="6.3.6"></a>
 ### 6.3.6 Security LCM
+Cloud Infrastructure LCM encompasses provisioning, deployment, configuration and management (resources scaling, services upgrades…) as described in [chapter 7](./chapter07.md). These operations must be securely performed in order to keep the infrastructure safe and operational.
+
+**Provisioning/Deployment**
+
+Regarding the provisioning of servers, switches, routers and networking, tools must be used to automate the provisioning eliminating human error. For Infrastructure hardware resources, a set of recommendations is detailed in [7.2.1](./chapter07.md#7.2.1) to automate and secure their provisioning.
+
+For OpenStack services and software components, deployment tools or components must be used to automate the deployment and avoid errors.  The deployment tool is a sensitive component storing critical information (deployment scripts, credentials…). 
+The following rules must be applied:
+
+- The boot of the deployment tool must be protected
```

Comment:
Do we mean the boot of the server or virtual machine hosting the deployment tool or are you recommending some other security wrapper around the software?

Reply:
@iangardner22 Yes I had in mind the boot of server or VM. Do you want to add more reco?

Reply:
Yes just a few words to show we are talking about the VM or Baremetal

Reply:
@iangardner22 Done!